specification/keyvault/data-plane/Microsoft.KeyVault/2016-10-01/keyvault.json

```diff
@@ -1220,14 +1226,15 @@
           "DeletedSecrets"
         ],
         "operationId": "PurgeDeletedSecret",
-        "description": "Permanently deletes the specified secret. aka purges the secret. Authorization: requires the secrets/purge permission.",
+        "summary": "Permanently deletes the specified secret.",
+        "description": "The purge deleted secret operation purges the secret. It requires the secrets/purge permission.",
```

Comment:
I would suggest a non-circular description: "The PurgeDeletedSecret operation removes a deleted secret permanently, without the possibility of recovery. The operation requires the 'secrets/purge' permissions, which must be assigned explicitly."